package.json

```diff
@@ -62,6 +62,7 @@
     "method-override": "^2.3.10",
     "mongoose": "4.7.4",
     "morgan": "1.9.0",
+    "multer": "false1.4.1",
```

Comment:
Not sure what false means here

Reply:
I have found yarn/npm doing this, I thought it was just me. `false` should be `^`